### PR TITLE
Revert #15697 — Change CSS Caret border-top from dashed to solid

### LINKED ISSUE #15682: .caret is smaller in Firefox than other browsers
Before and after: http://jsbin.com/daguhewoka/2/edit?html,css,output.

For fix change `solid` to `dashed` in https://github.com/twbs/bootstrap/blob/master/less/dropdowns.less#L13.

I believe that it is necessary to add a comment: `// dashed: fix caret size for Mozilla Firefox`.

### LINKED ISSUE #16657: .caret rendering problem in Firefox
caret_error
Link for demonstrating problem: http://jsbin.com/wifike/2/edit?html,css,output
This happend in Firefox 38.0.5 on Windows 7.
The border-top css value of caret in dropdown.less is "dashed" so it is displayed wrong in firefox. When i change the value to "solid" it is displayed right.

### PR DESCRIPTION
**Pull request by kkirsche**
Fix #16657

Reverts #1597 which fixed #15682

EDIT:
Is it worth adding a short inline comment why this is solid not dashed?

### CODE REVIEW

File: less/dropdowns.less [hunk 1]
```diff
@@ -10,7 +10,7 @@
   height: 0;
   margin-left: 2px;
   vertical-align: middle;
-  border-top:   @caret-width-base dashed;
+  border-top:   @caret-width-base solid;
```
Comment (cvrebert) [inline less/dropdowns.less]: This makes the next line unnecessary; please delete the now-redundant next line.

Comment (cvrebert) [inline less/dropdowns.less]: Also, please apply this fix to line 189 too.

Comment (kkirsche) [inline less/dropdowns.less]: Fixed redundant line(s) (one on this and one at 189) and made the change to line 189. Sorry about missing those

**Comment by mdo** [comment]
That other issue had a JS Bin example, was this tested in Windows and OS X versions of all browsers?

**Comment by kkirsche** [comment]
I don't have access to a windows machine to test this on, I have tested this in OS X on Firefox, Safari, Opera, and Chrome. In OS X though there is no noticeable difference between the two caret styles that I could tell in the latest versions of each of these browsers.

**Comment by cvrebert** [comment]
This is on hold due to the lack of confirmation in #16657.

**Comment by cvrebert** [comment]
Closing since we haven't been able to confirm/reproduce #16657.

**Pull request closed without merging**